core/src/main/scala/org/genivi/sota/core/VehicleUpdatesResource.scala

```diff
@@ -183,7 +183,9 @@ class VehicleUpdatesResource(db : Database, resolverClient: ExternalResolverClie
       put {
         path("installed") { updateInstalledPackages(vin) } ~
         path("order") { setInstallOrder(vin) } ~
-        (extractUuid & path("cancelupdate") ) { uuid => cancelUpdate(vin, uuid) }
+        (extractUuid & path("cancelupdate") ) { uuid =>
+          namespaceExtractor { ns => cancelUpdate(ns, vin, uuid) }
```

Comment:
True, client making request is a design consideration. For this particular endpoint however (`cancelupdate`) the only invoker seems to be the web app, and thus an `Option` is more palatable.